### DISCUSSION TITLE
海外银行证券开户薅羊毛小记 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#152] [Announcements]
# 海外银行证券开户薅羊毛小记 · BMPI

本文分享我在海外银行证券开户薅羊毛及被割韭菜的故事。

https://www.bmpi.dev/money/guide-to-open-sg-bank/

**Comment by achuan101** [comment]
ocbc不是华侨银行吗

**Reply by madawei2699** [reply]
是的，已经修正。

**Comment by Martin91** [comment]
:D，已经使用你的邀请码开通了，晚些转账进去。

**Comment by valuetodays** [comment]
请问，后续会有`开通证券账户、入金、买美股`这些操作的文章吗

**Reply by madawei2699** [reply]
暂时没这方面的写博客计划，不过会在我的投资助手会员社区里分享：https://www.myinvestpilot.com/
